docs/Demo_Projects/Tracking_tutorial.md

```diff
@@ -0,0 +1,388 @@
+# Tracking application - Thingy:91 to Power BI
+
+## Introduction
+Build a tracking application using the **Thingy:91** devkit, while leveraging the benefits of the LwM2M protocol and visualizing its data on **Microsoft Power BI**.
+
+## Prerequisites
+
+- [Thingy:91](https://www.nordicsemi.com/Products/Development-hardware/Nordic-Thingy-91) with SIM with access to LTE-M or NB-IoT networks
+- [Coiote IoT DM](https://eu.iot.avsystem.cloud/) account (premium)
+- [nRF Cloud](https://nrfcloud.com/) account
+- [nRF Connect for Desktop](https://www.nordicsemi.com/Products/Development-tools/nrf-connect-for-desktop)
+- [Microsoft Azure account](https://azure.microsoft.com/en-us/free/)
+- [Microsoft Power BI account](https://powerbi.microsoft.com/)
+
+## Architecture
+
+![Architecture tracking app](images/Architecture-tracking-demo.jpg "Anjay menuconfig")
+
+This tutorial uses the **Thingy:91** prototyping platform in combination with the **Coiote IoT Device Management** platform to build a cellular-connected tracking application. The integration with **nRF Cloud Locator** enables cell-based localization and optimizes the usage of the onboard GNSS. The location data, in combination with additional telemetry data is sent to **Azure IoT Hub** and visualized using **Microsoft Power BI**.
+
+## Part 1 - Connect the Thingy:91 to Coiote using the LwM2M Anjay client
+
+### Set up the Anjay Zephyr Client
+
+#### Get Zephyr and Python dependencies
+
+To get the Zephyr SDK and dependencies follow the first 4 steps of the instruction provided by [the Zephyr Project](https://docs.zephyrproject.org/latest/getting_started/index.html).
+
+0. [Select and update OS](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#select-and-update-os)
+0. [Install dependencies](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-dependencies)
+0. [Get Zephyr and install Python dependencies](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#get-zephyr-and-install-python-dependencies)
+0. [Install Zephyr SDK](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-zephyr-sdk)
+
+
+#### Clone the Anjay Zephyr repository
+
+Open the command line interface on your machine, then paste and run the following command:
+
+   ```
+   git clone https://github.com/AVSystem/Anjay-zephyr-client
+   ```
+
+### Compile the demo project
+
+0. Connect the Thingy:91 board to a USB port of your machine.
+0. Set West manifest path to `Anjay-zephyr-client/demo`, manifest file to `west-nrf.yml`, and run `west update`:
```

Comment:
How about changing the sentence a bit so that it ends with "...using the following commands"?

Reply:
Updated